src/main/kotlin/io/perfometer/itnernal/helper/parsers.kt

```diff
@@ -0,0 +1,12 @@
+package io.perfometer.itnernal.helper
```

Comment:
typo: internal

Reply:
Good catch! Strange, that I haven't noticed it in IDE spellcheck...